### ISSUE TITLE
Rule 3.2.2 requires small caps but Georgia does not have real small caps

### ISSUE THREAD

**Issue by bakert** [clagnut/webtypography#31]
TL; DR: we need a font that supports "real" small caps to proceed with the next item and so the site itself can follow the recommendations of the book. I suggest we use the open-licensed EB Garamond as the site's main font from now on.

Long version follows.

The relevant part of the text:

> 3.2.2 For abbreviations and acronyms in the midst of normal text, use spaced small caps. 
…
> Genuine small caps are not simply shrunken versions of the
> full caps. They differ from large caps in stroke weight, letterfit,
> and internal proportions as well as in height. Any good set of
> small caps is designed as such from the ground up. Thickening,
> shrinking and squashing the full caps with digital modification
> routines will only produce a parody. 

`font-family` values used in the site:

- `font-family`: 'premiera book', georgia, serif;
- `font-family`: 'premiera italic', georgia, serif;
- `font-family`: inconsolata, consolas, "andale mono", "droid sans mono", menlo, monospace; *(for code)*
- `font-family`: "univers bold condensed", impact, haettenschweiler, sans-serif; *(for one example)*

Preimiera Book seems to have been a font made by typejockeys that is no longer available? It is not included in the repo nor loaded into the site via @font-face or anything similar as far as I can see. I don't know if we have a copy, or if it has true small caps and old-style numbers, nor how it is licensed. As it stands Georgia will be the typeface that almost everyone is seeing the site in. Georgia does NOT have genuine small caps.

We can't present the site following the rules of the book without changing something about how we are using fonts. As the entire site is *about* typography it seemed worth discussing here before sending a PR.

Georgia Pro has genuine small caps but it is not free. It is licensed per-pageview – https://www.myfonts.com/fonts/ascender/georgia-pro/licensing.html

The book (v3 at least) is set in Minion Pro which is not available for the web under any license (Adobe will license it for desktop computers for $35-600 depending on exactly what parts you want but that doesn't help us).

Options:

– If we have a copy of the font and the right license use Premiera Book/Italic but inline a woff version or otherwise download it to the client rather than relying on them having it installed (very unlikely). I emailed typejockeys to see if they can clarify.
– License Georgia Pro. This is an ongoing cost for the host of the site. I'm not sure how much traffic it gets but I assume this is a bad idea.
– License another appropriate serif font with genuine small caps and old-style numbers that has a one-off fee. One of these probably – https://practicaltypography.com/minion-alternatives.html
– Pick an open source/free serif font with genuine small caps and old-style numbers.

This last option feels most promising as it will make including it in the GitHub repo a non-issue and won't cost anyone anything. Do we know of one that is good enough? EB Garamond seems like a top candidate – 

https://github.com/georgd/EB-Garamond
https://fonts.google.com/specimen/EB+Garamond 

?

In the course of writing/researching this I became mostly-convinced by the EB Garamond option so I'll try it out and send a PR.

**Comment by bakert** [comment]
I gave this a little outing on my local copy. I'm not sure how I feel about it. Italics are certainly different in EB Garamond than in Georgia. (OS X/Chrome.)

Georgia

<img width="1217" alt="screen shot 2018-11-30 at 14 46 32" src="https://user-images.githubusercontent.com/1541/49318800-d5028880-f4ae-11e8-9a83-16bcca040ac4.png">

EB Garamond with base font-size raised from 16px to 19px as it's much smaller at the same base size.

<img width="1221" alt="screen shot 2018-11-30 at 14 45 04" src="https://user-images.githubusercontent.com/1541/49318801-d5028880-f4ae-11e8-95e0-0fdafe62272a.png">

**Comment by bakert** [comment]
Here it is with the real small caps and all:

<img width="1304" alt="screen shot 2018-11-30 at 15 42 33" src="https://user-images.githubusercontent.com/1541/49320489-bbfdd580-f4b6-11e8-93ed-ca5042182dc8.png">

**Comment by bakert** [comment]
Closed via #32
